libs/core/src/lib/table/table.component.ts

```diff
@@ -118,6 +118,7 @@ export class TableComponent implements AfterViewInit, OnDestroy, FdTable {
     _onCellKeydown(event: KeyboardEvent, cell: TableCellDirective): void {
         const cellElement = cell.elementRef.nativeElement;
         if (KeyUtil.isKeyCode(event, [DOWN_ARROW, UP_ARROW])) {
+            event.preventDefault();
             const data = cell.getCellPosition();
             if (!data) {
                 return;
```

Comment:
I would suggest that if there's no data from getCellPosition, I would allow native behaviour (page scroll), so maybe move `event.preventDefault();` a bit down, after line 125